code/freespace2/freespace.cpp

```diff
@@ -1796,23 +1794,24 @@ void game_init()
 	cmdline_debug_print_cmdline();
 #endif
 
-	memset(whee, 0, sizeof(whee));
-
-	GetCurrentDirectory(MAX_PATH_LEN-1, whee);
-
-	strcat_s(whee, DIR_SEPARATOR_STR);
-	strcat_s(whee, EXE_FNAME);
+	base_path = SDL_GetBasePath();
+	if ( base_path == NULL ) {
+		SCP_Messagebox(MESSAGEBOX_ERROR, "Error finding base path");
+		exit(1);
+	}
 
 	profile_init();
 	//Initialize the libraries
 	s1 = timer_get_milliseconds();
 
-	if ( cfile_init(whee, strlen(Game_CDROM_dir) ? Game_CDROM_dir : NULL) ) {			// initialize before calling any cfopen stuff!!!
+	if ( cfile_init(base_path, strlen(Game_CDROM_dir) ? Game_CDROM_dir : NULL) ) {			// initialize before calling any cfopen stuff!!!
```

Comment:
`cfile_init` expects a path to a file, not a directory (`cfile.cpp:198` ends the string at the last directory separator). This means that linux and Windows will use the parent directory of the current working directory instead of the actual working directory.